Tools/TestWebKitAPI/Configurations/TestWebKitAPI.xcconfig

```diff
@@ -67,9 +67,18 @@ WK_VISIONKITCORE_LDFLAGS_iphonesimulator = -framework VisionKitCore;
 WK_VISIONKITCORE_LDFLAGS_xros = -framework VisionKitCore;
 WK_VISIONKITCORE_LDFLAGS_xrsimulator = -framework VisionKitCore;
 
-WK_OPENGL_LDFLAGS = $(WK_OPENGL_LDFLAGS_$(WK_COCOA_TOUCH));
-WK_OPENGL_LDFLAGS_ = -framework OpenGL;
-WK_OPENGL_LDFLAGS_cocoatouch = -framework OpenGLES;
+WK_OPENGL_LDFLAGS = $(WK_OPENGL_LDFLAGS_$(WK_PLATFORM_NAME));
+WK_OPENGL_LDFLAGS_macosx = -framework OpenGL;
+WK_OPENGL_LDFLAGS_maccatalyst = -framework OpenGL;
```

Comment:
the maccatalyst and ios family shouldn't link to opengl anymore

Reply:
What should we do about `WebGLNoCrashOnOtherThreadAccess`, in TestWebKitAPI? It's using EAGL on iOS and CGL on Catalyst. Seems like we need to remove it or rewrite it to use Metal in order to stop linking TestWebKitAPI against OpenGLES.

And since iOS is building without EAGL, I'm not sure why it's working at all.

Reply:
In the meantime, I'll have iOS TestWebKitAPI link against GLES to keep the queue green, but if there's an opportunity for easy clean up let's do it.